### DISCUSSION TITLE
Feature Request: adjustable size of Clock (Home and Screensaver)

### DISCUSSION THREAD

**Discussion by iamthe1st** [Blackymas/NSPanel_HA_Blueprint#1657] [Ideas]
Hi, i flashed my NsPanel today and i am very thankful for your hard work you have put into this!
I have one Feature Request: Please make the Size of the Clock adjustable.
I would prefer to see a bigger clock on my "home screen".

Also for Screensaver it would be perfect, if the Clock would be able to "fill" the screen.
Theank you!

**Comment by edwardtfn** [comment]
The challenge here is to add new fonts sizes to the TFT, as this is what takes more of the files space, making it larger and harder to transfer (and closer to the displays memory limits).
But I wanna take a look at this later. Could you please report it as an Enhancement request in the [Issues](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues) area?

**Comment by edwardtfn** [comment]
Moved to issues:
- #1689
